pkgs/development/rocm-modules/6/hiprand/default.nix

```diff
@@ -0,0 +1,76 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rocmUpdateScript
+, cmake
+, rocm-cmake
+, clr
+, rocrand
+, gtest
+, gbenchmark
+, buildTests ? false
```

Comment:
When this is set to true the build fails.

```
.2.drv^*'
-- Installing: /nix/store/wj6ad2h1mdq97vq51qxv5ps36l4imv6y-hiprand-6.0.2/bin/test_hiprand_cpp_wrapper
-- Set non-toolchain portion of runtime path of "/nix/store/wj6ad2h1mdq97vq51qxv5ps36l4imv6y-hiprand-6.0.2/bin/test_hiprand_cpp_wrapper" to ""
-- Installing: /nix/store/wj6ad2h1mdq97vq51qxv5ps36l4imv6y-hiprand-6.0.2/bin/test_hiprand_kernel
-- Set non-toolchain portion of runtime path of "/nix/store/wj6ad2h1mdq97vq51qxv5ps36l4imv6y-hiprand-6.0.2/bin/test_hiprand_kernel" to ""
-- Installing: /nix/store/wj6ad2h1mdq97vq51qxv5ps36l4imv6y-hiprand-6.0.2/bin/test_hiprand_linkage
-- Set non-toolchain portion of runtime path of "/nix/store/wj6ad2h1mdq97vq51qxv5ps36l4imv6y-hiprand-6.0.2/bin/test_hiprand_linkage" to ""
-- Installing: /nix/store/wj6ad2h1mdq97vq51qxv5ps36l4imv6y-hiprand-6.0.2/bin/hipRAND/CTestTestfile.cmake
-- Installing: /nix/store/wj6ad2h1mdq97vq51qxv5ps36l4imv6y-hiprand-6.0.2/share/doc/hiprand/LICENSE.txt
rmdir: failed to remove '/nix/store/wj6ad2h1mdq97vq51qxv5ps36l4imv6y-hiprand-6.0.2/bin': Directory not empty
/nix/store/hylcibqj87y8flsj51d2dpc7fiy6g1gx-stdenv-linux/setup: line 131: pop_var_context: head of shell_variables not a function context
```

Reply:
Looks like this is also the case for `rocrand`. :/

Reply:
Seems like it's trying to `rmdir` a non-empty directory, which is easy enough to fix. 

It's not empty because there's a .cmake file in a `hipRAND` directory:

> /nix/store/9ds4wynjsfzfcmbvyl74sygg6rmaiqih-hiprand-6.0.2/bin
> /nix/store/9ds4wynjsfzfcmbvyl74sygg6rmaiqih-hiprand-6.0.2/bin/hipRAND
> /nix/store/9ds4wynjsfzfcmbvyl74sygg6rmaiqih-hiprand-6.0.2/bin/hipRAND/CTestTestfile.cmake

which seems like it shouldn't be there. Changing to `rm -r`.

Reply:
Actually, I decided to `rm -r` the subdirectory inside of it and `rmdir $out/bin` so that it starts failing loudly again if there's more stuff added in there.

I've also updated the derivation for rocRAND, same story.